packages/lisk-api-client/package.json

```diff
@@ -41,21 +41,26 @@
 		"test:watch:min": "npm run test:watch -- --reporter=min",
 		"test:node":
 			"npm run build:check && BABEL_ENV=buildcheck mocha test --compilers js:babel-register --recursive",
-		"test:browser": "cypress run --env ROOT_DIR=\"${PWD##*/}\"",
-		"serve:browsertest": "http-server browsertest",
+		"pretest:browser":
+			"npm run build:browsertest && http-server -p 11541 browsertest &",
+		"test:browser":
+			"wait-on http://localhost:11541 && cypress run --env ROOT_DIR=\"${PWD##*/}\" && echo 'test-finished'",
+		"posttest:browser": "echo 'posttest' && kill $(lsof -t -i:11541)",
 		"cover":
 			"if [ -z $JENKINS_HOME ]; then npm run cover:local; else npm run cover:ci; fi",
 		"cover:base": "NODE_ENV=test nyc report",
 		"cover:local": "npm run cover:base -- --reporter=html --reporter=text",
 		"cover:ci": "npm run cover:base -- --reporter=text",
 		"build:browsertest":
 			"npm run transpile:browsertest && npm run browserify:browsertest && npm run uglify:browsertest",
-		"postbuild:browsertest": "rm -r browsertest/src browsertest/test",
+		"postbuild:browsertest":
+			"rm -r browsertest.build/src browsertest.build/test",
 		"prebuild:node": "rm -r dist-node/* || mkdir dist-node | echo",
 		"build:node": "npm run transpile",
 		"prebuild:browser":
 			"rm ./dist-browser/index.js ./dist-browser/index.min.js | echo",
-		"build:browser": "npm run browserify && npm run uglify",
+		"build:browser":
+			"npm run build:node && npm run browserify && npm run uglify",
 		"prebuild": "npm run prebuild:node && npm run prebuild:browser",
 		"build": "npm run build:node && npm run build:browser",
```

Comment:
If `build:browser` includes `build:node` we don't need it in `build` as well.

Reply:
@willclarktech I think it still makes sense to keep it since it will be browser distribution